compat/mingw.c

```diff
@@ -225,6 +225,8 @@ static int retry_ask_yes_no(int *tries, const char *format, ...)
 		return 1;
 	}
 
+	if (!*format)
```

Comment:
The function is called `retry_ask_yes_no`. Here you add a hack to skip the asking part. If you need a function to retry without asking, then please give it an appropriate name such as `retry`.

Anyway, there is no reason to behave differently from the unlink. Please ask the user if they want to retry.

Reply:
Removed